### PR TITLE
{vis}[GCC/8.2.0-2.31.1] wxWidgets v3.0.4

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Depends on ~~#8118~~, ~~#8124~~, ~~#8120~~

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/fcf920a5b04ee4d46f2b37abe5e6aefa for a full test report.

**Comment by akesandgren** [comment]
For some reason this fails to build in my production env.... so don't rush with merging it.

**Comment by akesandgren** [comment]
Never mind, I had forgotten to rebuild WebKitGTK+ on the production tree after the upgrade to GStreamer 1.16.0

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3118.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/ed69471b620f8fbd3c64476adb3a2e99 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/cc486c9dce175f7c847e39cd1ecc249d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2039.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/398706a47fd93ae036583b636355cbb4 for a full test report.

File: easybuild/easyconfigs/w/wxWidgets/wxWidgets-3.0.4-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    ('GTK+', '3.24.8'),
+    ('Mesa', '19.0.1'),
+    ('libGLU', '9.0.0'),
+    ('SDL2', '2.0.9'),
+    ('GST-plugins-base', '1.16.0'),
+    ('GLib', '2.60.1'),
+    ('WebKitGTK+', '2.24.1'),
+]
+
+preconfigopts = 'CPATH=$CPATH:$EBROOTGTKPLUS/include/gtk-3.0:$EBROOTGLIB/include/glib-2.0:$EBROOTGLIB/lib/glib-2.0/include '
```
Comment (boegel) [inline easybuild/easyconfigs/w/wxWidgets/wxWidgets-3.0.4-GCC-8.2.0-2.31.1.eb]: @akesandgren line too long

File: easybuild/easyconfigs/w/wxWidgets/wxWidgets-3.0.4-GCC-8.2.0-2.31.1.eb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+configopts = '--enable-intl --enable-ipv6 '
+# Options required by wxPython
+configopts += '--with-gtk=3 --with-gtk-prefix=$EBROOTGTKPLUS '
+configopts += '--with-opengl '
+configopts += '--enable-unicode --enable-sound --enable-graphics_ctx '
+configopts += '--enable-mediactrl --enable-display --enable-geometry '
+configopts += '--enable-debug_flag --enable-optimise --disable-debugreport '
+configopts += '--enable-autoidman --with-sdl '
+
+prebuildopts = 'CPATH=$CPATH:$EBROOTGTKPLUS/include/gtk-3.0:$EBROOTGLIB/include/glib-2.0:$EBROOTGLIB/lib/glib-2.0/include '
```
Comment (boegel) [inline easybuild/easyconfigs/w/wxWidgets/wxWidgets-3.0.4-GCC-8.2.0-2.31.1.eb]: @akesandgren line too long

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
node2404.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7a60fc3519f28b92e70a4360b6ff96eb for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/7bbce3b88d5de593428f7f08b2c88c91 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/5a17d5f6eacf0ad54499e43eea7773d2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/036b191bdaff8665d5fee7b508d4b443 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2512.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0a5762832115b72327a3f8b8c94793b3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**